docs/en/observability/cloud-monitoring/aws/monitor-amazon-sqs.asciidoc

```diff
@@ -0,0 +1,140 @@
+[[monitor-amazon-sqs]]
+= Monitor Amazon Simple Queue Service (SQS)
+
+++++
+<titleabbrev>Monitor SQS</titleabbrev>
+++++
+
+The main parts in a distributed messaging system include the software components
+of your system, the queue (distributed on Amazon SQS), and the messages in the
+queue.
+
+Amazon Simple Queue Service (SQS) is a message queuing service that makes it
```

Comment:
Add generic link to info about the service. Make sure the link is "future proof" (unlikely to change)